client/source/components/questionsAnswers/LoadAddQA.jsx

```diff
@@ -14,10 +46,16 @@ const LoadAddQA = function CreateLoadAddQAComponent({
         }}
       >More Answered Questions
       </button>
-      <button type="button">Add a Question</button>
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalFormTitle
```

Comment:
This new ModalFormTitle looks great!

Reply:
Thank you!

Reply:
Also, I moved it to the shared/form folder so you can use it too!